### DISCUSSION TITLE
Error response from daemon

### DISCUSSION THREAD

**Discussion by ricardovr22** [containrrr/watchtower#663] [Q&A] [answered]
Hi
First of all, thanks for the container, it is really helpful
I am having been using the container without any problems for the last month. However, for 5 days ago, im unable to update my images. In all the cases i get the following error "Error response from daemon... Client.Timeout exceeded while awaiting headers). Proceeding to next"
I use docker on a synology
Thanks for any help

**Comment by piksel** [comment, marked as answer]
Unfortunately, that is beyond watchtowers control. It means that the docker daemon cannot communicate with the docker hub. It might be due to some network issues or configuration perhaps?

**Reply by simonaronsson** [reply]
If the origin of the error is state-based, you're usually able to solve it by either rebooting the docker service or the full host OS.
